app/code/core/Mage/Usa/Model/Shipping/Carrier/Ups.php

```diff
@@ -1330,18 +1330,14 @@ public function getResponse()
     public function getAllowedMethods()
     {
         $allowedMethods = explode(',', (string)$this->getConfigData('allowed_methods'));
-        $isUpsXml = $this->getConfigData('type') === 'UPS_XML';
-        $isUpsRest = $this->getConfigData('type') === 'UPS_REST';
         $origin = $this->getConfigData('origin_shipment');
```

Comment:
I would remove this line and merge it with the next one but at least let's remove the empty line below